Interview Questions

```diff
@@ -0,0 +1,78 @@
+# Interview questions for Junior Front-End Web Developers
+
+The purpose of this article is to familiarize fresh Front-End Web Developers with text book questions. If you are just getting into the developer world there is a good chance you will feel nervous when interviewing for a developer's positon. Because of this, some relatively basic questions could feel more difficult then how they actually are. A good counter for this scenario is to be properly prepared. Without further ado, here are a few questions that you could encounter:
+
+1)	Can you explain the meaning of “inline-block”?  
+
+
+
+2)	If you have and ul element with several li elements, how would you go about setting up these elements in a row by using HTML and CSS? 
+
+
+3)	What is the meaning of floats? How do they work? When would you use floats?
+
+
+4)	When would you use:  clear: both;?
+
+
+5)	What is Bootstrap, how does it work and why did you use it?
```

Comment:
What is the interviewee has never used Bootstrap? You could change the last line to 

> When would you use it?

or 

> When do you think it would be beneficial to writing your styling rules with Bootstrap?